<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticsWindow</class>
 <widget class="QDialog" name="StatisticsWindow">
 <property name="styleSheet">
 <string notr="true">
 /* ОКНО СТАТИСТИКИ - ДОПОЛНИТЕЛЬНЫЕ СТИЛИ */
 #tableWidget {
     alternate-background-color: #FAFAFF;
     font-size: 11pt;
 }

 #tableWidget QTableCornerButton::section {
     background-color: #EDF4F8;
 }

 #calculateButton {
     background-color: #F1E8D4;
 }

 #compareButton {
     background-color: #DCE6C8;
 }

 #totalLabel {
     font-weight: 600;
     font-size: 12pt;
     color: #5A5A5A;
     padding: 10px;
     background-color: #F0F5F9;
     border-radius: 8px;
 }
 </string>
 </property>
  <property name="windowTitle">
   <string>Статистика расходов</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Период 1:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="period1Start"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>по</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="period1End"/>
     </item>
     <item>
      <widget class="QPushButton" name="calculateButton">
       <property name="text">
        <string>Рассчитать</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Период 2:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="period2Start"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>по</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="period2End"/>
     </item>
     <item>
      <widget class="QPushButton" name="compareButton">
       <property name="text">
        <string>Сравнить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="totalLabel">
     <property name="text">
      <string>Итого: 0 руб.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
